src/definitions_cxx.hpp

```diff
@@ -1052,6 +1052,7 @@ constexpr int32_t NUM_CHANNELS = 18;
 constexpr int32_t MIDI_CHANNEL_NONE = 255;
 
 constexpr int32_t IS_A_CC = NUM_CHANNELS;
+constexpr int32_t IS_A_PC = 128; // CC128 is max.
```

Comment:
That makes sense. I'm silly, i did 128 because that's the maximum CC number because initially... I wanted the hack for using out of bounds for PC to be separate.. or unconflatable from the out of bounds channel strategy used for CC where we call note with a channel > 16. I'm actually on the fence somewhat between the two approaches because the approach i did choose, even if i fix the IS_A_PC values... still feels like we could approach it differently and help downstream consumers of this weird way of doing things account for less complexity? 

Thanks for pointing it out, will fix.

Reply:
we should write an issue for this, with my novice level of experience everything takes me 10x longer and i'm 10x more likely to make a silly mistake and i've already bitten off more than i can chew in other areas of the codebase. However, I'm working extremely heavily with midi messages and have observed some consistency in terms of common APIs for dealing with midi data across 3-4 different codebases and libraries/languages.